.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
+
+- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] My pull request targets the `master` branch of creativecommons.github.io-source.
```

Comment:
I would add another item to ensure that the person making the pull request has tried building the site using Lektor locally and verified that there are no build errors.

Reply:
Just added another bullet point to the template.